Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,16 +1,53 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Globalization;
 using System.IO;
-using System.Linq;
 using System.Text;
 
 namespace BootCamp.Chapter
 {
-    class Program
+    internal class Program
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
+            var contents = FileCleaner.Clean(@"Input\Balances.corrupted", @"Input\Balances.clean");
+            
+
+        }
+
+       
+
+
+        private static void FindStaticalData()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'FindStaticalData'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)